src/main/java/com/notably/logic/commands/EditCommand.java

```diff
@@ -16,15 +19,20 @@
     private static final String ERROR_ROOT_MODIFICATION = "Editing the root block is forbidden.";
     private static final AbsolutePath ROOT = AbsolutePath.fromString("/");
 
+    private final Logger logger = LogsCenter.getLogger(getClass());
```

Comment:
Oops sorry missed this before. Just a tiny thing. Let's make this `logger` a `private static final` field since we don't need to create one for each instance of `EditCommand`!